code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm

```diff
@@ -218,8 +218,16 @@
 	see_in_dark = 8
 	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
 
+//Tendril-spawned Legion remains, the charred skeletons of those whose bodies sank into laval or fell into chasms.
+/obj/effect/mob_spawn/human/corpse/charredskeleton
+	name = "charred skeletal remains"
+	mob_name = "ashen skeleton"
+	mob_gender = NEUTER
+	husk = FALSE
+	mob_species = /datum/species/skeleton
```

Comment:
Right, that's a good point. I originally had burn damage on them, but for some reason that gave them the husk sprite rather than the skeletons. I'll look through mob code and see if I can exlude skeletons from husking.